app/views/users/show.html.erb

```diff
@@ -1,25 +1,29 @@
-<section class='single-user-container'>
-  <div class='single-user-content'>
-    <img src="<%= @user.photo %>" alt="picture of user" class='single-user-img'>
-    <div class='single-user-text'>
-      <p class='single-user-name'><%= @user.name %></p>
-      <p class='single-user-post-numba'>Number of posts: <%= @user.posts_counter %></p>
-    </div>
-  </div>
-  <p class='single-user-bio'>Bio: <%= @user.bio %></p>
+<% #This is for a single user %>
 
-  <ul class='single-user-post-container'>
-    <% @user.recent_posts.each_with_index do |post, index| %>
-      <li class='single-user-post-content'>
-        <p>Post <%= index + 1 %></p>
-        <%= link_to post.title, user_post_path(@user, post) %>
-        <p><%= post.text %></p>
-        Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
-      </li>
-    <% end %>
-  </ul>
+<section class= 'single-user-container'>
+    <div class='single-user-content'>
+      <img src="<%= @user.photo %>" alt="picture of user" class='single-user-img'>
+      <div class= 'single-user-text'>
+        <p class='single-user-name'><%= @user.name %></p>
+        <p class= 'single-user-post-numba'>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+      </div>
+      <p class= 'single-user-bio'>Bio: <%= @user.bio %></p>
+    
+    <ul class='single-user-post-container'>
+      <% @user.recent_posts.each_with_index do |post, index| %>
+        <li class='single-user-post-content'>
+         <p>Post <%= index + 1 %></p>
+         <%= link_to post.title, user_posts_path(@user)%>
+         <p><%= post.text %></p>
+          Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
```

Comment:
- [ ] Kindly take note of the 7 failed tests. You did a good job, but les fix this so you get approved 👍 

### View:
![image](https://github.com/Kingno3l/Ruby-Blog-App/assets/50990925/a92f6c3e-3545-4aba-86bf-650638a4d035)